plugins/woocommerce/src/Blocks/BlockTypes/CollectionAttributeFilter.php

```diff
@@ -179,7 +179,7 @@ function( $term ) use ( $attribute_counts, $selected_terms ) {
 			$attribute_terms
 		);
 
-		$filter_content = 'dropdown' === $attributes['displayStyle'] ? $this->render_attribute_dropdown( $attribute_options, $attributes ) : $this->render_attribute_list( $attribute_options, $attributes );
+		$filter_content = 'dropdown' === $attributes['displayStyle'] ? $this->render_attribute_dropdown( $attribute_options, $attributes ) : $this->render_attribute_checkbox_list( $attribute_options, $attributes );
```

Comment:
We're passing `$attributes` array but used it as `$show_counts` boolean value in the `render_attribute_checkbox_list` method body.

Reply:
whoops! thanks for spotting

Reply:
It felt inconsistent to have both methods take different sigs so for consistency they both take attributes now and we determine from `$attributes['showCounts']` whether to show counts. https://github.com/woocommerce/woocommerce/pull/43217/commits/223c796e2f9fd164084a4610906bc6bf224eb486